fft/src/KokkosFFT_ROCM_types.hpp

```diff
@@ -111,9 +112,10 @@ struct ScopedRocfftExecutionInfo {
 
     // Set work buffer
     if (workbuffersize > 0) {
-      m_buffer = BufferViewType("workbuffer", workbuffersize);
-      status   = rocfft_execution_info_set_work_buffer(
-          m_execution_info, (void *)m_buffer.data(), workbuffersize);
+      hipError_t hip_status = hipMalloc(&m_workbuffer, workbuffersize);
```

Comment:
What do you think of using `Kokkos::kokkos_malloc` ? This would allow to trace the memory allocation in Kokkos tools.

Reply:
That may be a good idea. I will replace this with `Kokkos::kokkos_malloc`.

Reply:
Sure
I will try that.